README.md

```diff
@@ -9,9 +9,9 @@ To use these tools, obtain Azure Stack compatible Azure PowerShell module. Unles
 For PowerShell, install the following:
 
 ```powershell
-Install-Module -Name 'AzureRm.Bootstrapper'
-Install-AzureRmProfile -profile '2017-03-09-profile' -Force
-Install-Module -Name AzureStack -RequiredVersion 1.2.10
+Install-Module -Name 'AzureRm.Bootstrapper'
```

Comment:
what happens when there is an existing version of  bootstrapper

Reply:
@viananth

Reply:
The default behavior of Install-Module is to install the latest available version (if it's not already installed). If a previous version is installed, it will simply install the new version